docs/_sass/_main.scss

```diff
@@ -2,6 +2,16 @@
 @import 'colors';
 @import 'fonts';
 
+$color-appBG: $color-green-appBG;
+$color-highlightBG: $color-green-highlightBG;
+$color-borders: $color-green-borders;
+$color-icons: $color-green-icons;
+$color-blue: $color-blue200;
+$color-text: $color-white;
+$color-link: $color-link;
```

Comment:
Actually yeah, this one would be blue200a

Reply:
@shawnborton I think $color-link should be blue300 or blue400 because it's darker

Reply:
That's not what we're using for our theme colors. Can you please make it 200?

Reply:
I'm not sure if I am following? 

We just need `blue200` and `blue100` in `colors.css`

Then we need `link` and `link-hovered` in `main.scss`

Reply:
@shawnborton I just updated PR, please help to check and ping me if have any other problem

Reply:
Sorry, to clarify:
Link = blue200 = `#5AB0FF`
Link hovered = blue100 = `#8DC8FF`

Reply:
@shawnborton Actually now color of link hover is `#B0D9FF` do you want to change it to `#8DC8FF` that is blue100

Reply:
I don't understand your question?

Reply:
@shawnborton I think blue100 should be `#B0D9FF` because it's color when hover the link now

Reply:
Ah, you are right - sorry for the confusion. So it looks like this is correct:

Link = blue300 = #5AB0FF
Link hovered = blue200 = #8DC8FF

Sorry for all of the back and forth!

Reply:
@shawnborton No problem. I just update your review in PR please help to check again